.github/workflows/main.yml

```diff
@@ -36,7 +36,7 @@ on:
       platforms:
         description: 'Platform(s) to execute on (comma separated, e.g. "linux-x64, macos, aarch64")'
         required: true
-        default: 'linux-x64, linux-x86-hs, linux-x64-variants, linux-cross-compile, alpine-linux-x64, macos-x64, macos-aarch64, windows-x64, windows-aarch64, docs'
+        default: 'linux-x64, linux-x64-variants, linux-cross-compile, alpine-linux-x64, macos-x64, macos-aarch64, windows-x64, windows-aarch64, docs'
```

Comment:
Why does deprecation imply we stop caring about it completely? Shouldn't we continue to test until it is removed?

Reply:
I think in reverse: we have already stopped caring about 32-bit x86 completely, and now our build/test configs are catching up with reality :) 

We added 32-bit x86 GHA testing when there were active maintainers, and so to avoid simple breakages that help maintenance, at the expense of folks discovering and fixing build failures there. Today, there are no maintainers who are willing to continue. So, now saddling everyone with 32-bit x86 code build issues helps no one, and fixing issues in the code that is going away would be just extra work for no real gain.

Whoever -- if anyone -- cares about 32-bit x86 port after this, would need to build and test the port themselves.